test/clj/rems/test_browser.clj

```diff
@@ -901,9 +917,30 @@
         (btu/fill-human :fields-0-title-sv "Description (SV)")
 
         (btu/scroll-and-click :save)
-        (btu/wait-visible {:tag :h1 :fn/text "Form"})
         (btu/wait-page-loaded)
-        (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))))
+        (btu/scroll-query-el (first (btu/query-all {:tag :label :class :application-field-label})) {"block" "center"})
```

Comment:
Why does this need the scroll?

Reply:
Even so you don't need to use the `-el` version to write this. It could be simply
`(btu/scroll-query {:class :application-field-label})`?